### ISSUE TITLE
Can you specify your licence ? 

### ISSUE THREAD

**Issue by mikefaille** [synereo/synereo#3]
By default, in Canada, we can't checkout code without any licence by law.

If it's a Free Libre and Open Source Software (FLOSS) please specify your licence to maximise community contribution for this great project. 

I suggest to use protective licence for the software like Syncthing : 
https://github.com/syncthing/syncthing/blob/master/LICENSE

Then for protocol, I suggest you to use more liberal licence.
https://github.com/syncthing/specs

**Comment by henrytill** [comment]
Thanks for your interest and for bringing this up.  We hope to have this situation remedied soon.

The current state of things:

GLoSEval is licensed Apache v2.0:
https://github.com/synereo/synereo/blob/staging/gloseval/LICENSE

Agent Service is in the process of transitioning to Apache v2.0.  Relicensing has occurred in the SOC-91 branch and will be soon merged back into `staging` and `master`:
https://github.com/synereo/synereo/blob/SOC-91/agent-service/LICENSE

I've opened an issue on our JIRA tracker about licensing SpecialK:
https://synereo.atlassian.net/browse/SOC-300

**Comment by mikefaille** [comment]
Allright. This thread will help others then. Ty.

**Comment by henrytill** [comment]
Ty!  I'll post updates here as the situation improves...

**Comment by henrytill** [comment]
@mikefaille: I believe this issue has been remedied in https://github.com/synereo/synereo/commit/f66264c724396c7de440a3042ba0b5eb0665b7d6.

**TL;DR**: All three projects now have Apache v2.0 Licenses.
